resolwe/flow/executors/__main__.py

```diff
@@ -28,14 +28,41 @@
 
 import argparse
 import asyncio
+import logging
+import sys
+import traceback
 from importlib import import_module
 
 from . import manager_commands
-from .collect import collect_files
-from .global_settings import DATA
+from .global_settings import DATA, DATA_META
 from .logger import configure_logging
-from .protocol import ExecutorFiles
-from .transfer import transfer_data
+from .protocol import ExecutorProtocol
+
+
+def handle_exception(exc_type, exc_value, exc_traceback):
+    """Log unhandled exceptions."""
+    message = "".join(traceback.format_exception(exc_type, exc_value, exc_traceback))
+    logger.error("Unhandled exception in executor: {}".format(message),)
```

Comment:
Would it make sense to put a new line after colon?

Reply:
Yes, and I have removed colon too.